lib/index.js

```diff
@@ -13,7 +13,9 @@ function createJSONRules() {
         '*': (x, y, acc) => acc * y,
         '%': (x, y, acc) => acc % y,
         includes: (arr, value) => {
-            if (Array.isArray(arr)) {
+            if (!arr) {
```

Comment:
`!arr` will be truthy for `false`, `0`, `''`, `undefined`, `NaN`, and `null`. Is this the desired behaviour?

 Would it be better to explicitly check for it not being an array (!Array.isArray(arr)!)? I am not too familiar with this code. Just throwing a question out there!